src/app.js

```diff
@@ -0,0 +1,42 @@
+//Importing required packages
+const express = require("express");
+const cors = require("cors");
+const bodyParser = require("body-parser");
+require('dotenv').config()
+const newsApi = require('./routes/news');
+const authApi = require('./routes/auth');
+const userApi = require('./routes/user');
+
+const verifyToken = require('./middleware/authJWT');
+const routes = express.Router();
+
+//initializing app
+const app = express();
+
+//adding required middlewares
+app.use(cors());
+app.use(routes);
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
This is fine but I think bodyparser is depreciated now and is included in express itself now.